app/models/badges/greedo.rb

```diff
@@ -0,0 +1,19 @@
+class Badge::Greedo
+  BADGE_ID = 6
+  DISPLAY_NAME = 'Greedo'
+  SLUG = 'greedo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.